4.analyze_data/notebooks/UMAP/0.UMAP_coordinates.ipynb

```diff
@@ -59,20 +59,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 5,
```

Comment:
looks like this cell was executed out of order. (reminder to make sure cells are sequentially executed! 😄 )

Reply:
Ahh great catch! I'll fix prior to merging!